Bash missing in Pterodactyl Docker image

The latest Pterodactyl (Arm64) Docker image fails with the following error: `Error: Cannot find module '/bin/bash'`

<details>
<summary>Logs</summary>

```
[Pterodactyl Daemon]: Checking server disk space usage, this could take a few seconds...
[Pterodactyl Daemon]: Updating process configuration files...
[Pterodactyl Daemon]: Ensuring file permissions are set correctly, this could take a few seconds...
container@pterodactyl~ Server marked as starting...
[Pterodactyl Daemon]: Pulling Docker container image, this could take a few minutes to complete...
Pulling from dynamicabot/dynamica-v2 
Status: Image is up to date for ghcr.io/dynamicabot/dynamica-v2:latest-pterodactyl 
Digest: sha256:7cd46cecd180da6e34c0d82464cca6bec4bddd66e0b879bc8c75797bdcf8ef11 
[Pterodactyl Daemon]: Finished pulling Docker container image
node:internal/modules/cjs/loader:1145
  throw err;
  ^
Error: Cannot find module '/bin/bash'
    at Module._resolveFilename (node:internal/modules/cjs/loader:1142:15)
    at Module._load (node:internal/modules/cjs/loader:983:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:142:12)
    at node:internal/main/run_main_module:28:49 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
Node.js v21.7.3
container@pterodactyl~ Server marked as offline...
[Pterodactyl Daemon]: ---------- Detected server process in a crashed state! ----------
[Pterodactyl Daemon]: Exit code: 1
[Pterodactyl Daemon]: Out of memory: false
[Pterodactyl Daemon]: Aborting automatic restart, last crash occurred less than 60 seconds ago.
```

</details>

--- a/.changeset/witty-bottles-shop.md
+++ b/.changeset/witty-bottles-shop.md
@@ -1,0 +1,5 @@
+---
+"dynamica-v2": patch
+---
+
+Fix issue with bash not existing in the pterodactyl container by switching to sh.

--- a/Dockerfile
+++ b/Dockerfile
@@ -29,7 +29,7 @@ ENV DATABASE_URL "file:/home/container/dynamica/db.sqlite"
 ARG VERSION
 ENV VERSION=$VERSION
 
-RUN adduser -H -D container -s /bin/bash
+RUN adduser -H -D container -s /bin/sh
 ENV  USER=container HOME=/home/container
 USER container
 
@@ -38,7 +38,7 @@ HEALTHCHECK  --interval=5m --timeout=3s \
 
 COPY entrypoint.sh /entrypoint.sh
 
-CMD [ "/bin/bash", "/entrypoint.sh" ]
+CMD [ "/bin/sh", "/entrypoint.sh" ]
 
 # Default Image
 FROM base

